test/templates/list-group/index.njk

```diff
@@ -13,19 +10,19 @@
 {% endset %}
 {{ format(example) }}
 
-<br><br>
+<br /><br />
 <h2>with actions</h2>
 {% set example %}
 {{ list_group(data.properties) }}
 {% endset %}
 {{ format(example) }}
 
-<br><br>
+<br /><br />
 <h2>with checkboxes</h2>
 {% set example %}
 {{  list_group(
       properties={
-          items: [form_item("checkbox",{ label: "List item 1" }),form_item("checkbox",{ label: "List item 2 sit amet consectetur adipisicing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua" }),form_item("checkbox",{ label: "List item 3" })]
+          items: ["<div class='test-pg__checkbox'></div>I have a bike","<div class='test-pg__checkbox'></div>I have a motobike","<div class='test-pg__checkbox'></div>I have a car"]
```

Comment:
Looks like we can just get rid of the checkbox examples as they have no impact on the list group styles.